packages/frontend/src/pages/Inbox/Inbox.tsx

```diff
@@ -269,6 +265,10 @@ export const Inbox = ({ viewType }: InboxProps) => {
         />
       )}
       <section>
+        {isSearchInProgress ?
+          <p>Spinner</p> :
+          !isSearchInProgress && !!searchString?.length && <h2>{t('search.search.results', { count: searchResults.length })}</h2>}
```

Comment:
Ikke så lett logikk å lese her hvor en har to lag

Først sjekker den om `isSearchInProgress` og viser en spinner.
om den er false så sjekker den på nytt at den er false og at searchString?.length er true før den viser hvor mange søkeresultater det er..

Og vil ikke det is at om det er 0 resultater så bare forsvinner teksten med hvor mange resultater det er?

Reply:
![CleanShot 2024-06-13 at 13 25 36](https://github.com/digdir/dialogporten-frontend/assets/144693023/4576d08c-5437-40a1-a911-5cdaf6f81b36)